mempool/v0/reactor_test.go

```diff
@@ -64,13 +64,15 @@ func TestReactorBroadcastTxsMessage(t *testing.T) {
 
 	txs := checkTxs(t, reactors[0].mempool, numTxs, mempool.UnknownPeerID)
 	waitForTxsOnReactors(t, txs, reactors)
+
+	leaktest.CheckTimeout(t, 10*time.Second)()
```

Comment:
I don't think you need this line because we call `leaktest.CheckTimeout(...)` in `t.Cleanip`

Reply:
we didn't put it in `t.Cleanup` in the reactor_test. But yes, I will move it to `t.Cleanup`